src/applications/personalization/dashboard/tests/e2e/loa1.cypress.spec.js

```diff
@@ -111,3 +111,14 @@ describe('The My VA Dashboard', () => {
     loa1DashboardTest(true, stubs);
   });
 });
+
+describe('When clicking on the verify your identity link', () => {
+  it('should focus on the h1 element', () => {
+    cy.login(loa1User);
+    cy.visit(manifest.rootUrl);
+    cy.findByRole('link', { name: /verify your identity/i })
+      .should('exist')
+      .click();
+    cy.get('h1').should('be.focused');
```

Comment:
I normally hate `cy.get()`. But in this case it absolutely does the job since there is only one `h1` on the page and you didn't pass any additional CSS class names or selectors.

Reply:
The alternative approach, which might be a teeny tiny bit better would be `cy.findByRole('heading', {level: 1})`. The biggest reason that'd be better is so that we are consistent with using Testing Library queries. One less use of `cy.get()` is a win. The other, better, reason it might a bit better is on the off chance that someone did something really stupid and somehow broke the `h1`'s role. Using `findByRole('heading')` would fail because it wouldn't find a thing with the accessible role of heading. I saw something like this happen with a link tag once. The tag was effectively broken and didn't function as a link, and if a test had been trying to select it with `ByRole('link')` the test would have failed.

Reply:
Interesting, @erikphansen! We're using `cy.get` all over the place. I'm curious what other objections you have to it, in addition to the role issue?